doc/latex.rst

```diff
@@ -900,9 +900,8 @@ macros may be significant.
                                ``attentionBorderColor``, ``dangerBorderColor``,
                                ``errorBorderColor``
 
-.. |wgbdcolorslatex| replace:: ``warningBorderColor``, ``cautionBorderColor``,
-                               ``attentionB..C..``, ``dangerB..C..``,
-                               ``errorB..C..``
+.. |wgbdcolorslatex| replace:: ``warningBorderColor``, and 
+                               ``(caution|attention|danger|error)BorderColor``
```

Comment:
one will not find by search attentionBorderColor, but it was not the case earlier either